src/components/alarm_definitions.js

```diff
@@ -22,9 +22,9 @@ import MonascaClient from './monasca_client';
 export class AlarmDefinitionsPageCtrl {
 
   /** @ngInject */
-  constructor($scope, $injector, backendSrv, datasourceSrv, alertSrv) {
+  constructor($scope, $injector, backendSrv, datasourceSrv, alertSrv, monascaClientSrv) {
```

Comment:
nit: Since this change makes backendSrv, datasourceSrv redundant it would be better to remove them in this commit rather than the next one.